src/button/button.scss

```diff
@@ -35,7 +35,7 @@ $iui-button-padding-large: $iui-xs * 6;
     background-color $iui-speed-fast ease-out,
     border-color $iui-speed-fast ease-out;
 
-  > .iui-icon {
+  > iui-icon {
```

Comment:
Did you forgot dot? Because now it is looking for `<iui-con>` HTML element.

Reply:
Who needs dot anyway 😅